src/utils/testsToStatusUpdate.ts

```diff
@@ -1,57 +1,54 @@
-import type { API } from 'storybook/internal/manager-api';
-import type { API_StatusUpdate, API_StatusValue, StoryId } from 'storybook/internal/types';
+import { type Status, type StatusValue, type StoryId } from 'storybook/internal/types';
 
-import { PANEL_ID } from '../constants';
+import { ADDON_ID } from '../constants';
 import { StatusTestFieldsFragment, TestStatus } from '../gql/graphql';
 
-export const statusMap: Partial<Record<TestStatus, API_StatusValue>> = {
-  [TestStatus.Pending]: 'warn',
-  [TestStatus.Failed]: 'error',
-  [TestStatus.Denied]: 'error',
-  [TestStatus.Broken]: 'error',
+export const statusMap: Record<TestStatus, StatusValue> = {
+  [TestStatus.Pending]: 'status-value:warning',
+  [TestStatus.Failed]: 'status-value:error',
+  [TestStatus.Denied]: 'status-value:error',
+  [TestStatus.Broken]: 'status-value:error',
+  [TestStatus.InProgress]: 'status-value:pending',
+  [TestStatus.Accepted]: 'status-value:success',
+  [TestStatus.Passed]: 'status-value:success',
```

Comment:
We should talk about this. Essentially I think this change makes it so that passing statuses returned from the API will now get green check marks in the sidebar, while before they would just not show up at all.